docs/scripts/fetch_contributors.py

```
@@ -0,0 +1,104 @@
+    r"^renovate",
+    r"^github-actions",
+    r"^semantic-release",
+    r"^imgbot",
+    r"^codecov",
+    r"^snyk",
+    r"^greenkeeper",
+    r"^depfu",
+    r"^pyup-bot",
+]
+
+def is_bot(username):
```

Ruff (0.12.2)

Missing return type annotation for public function `is_bot`

(ANN201)